bad_code.java

```diff
@@ -8,14 +8,15 @@ public static void main(String[] args) {
 }
 
 class ParentClass {
-    public int someNumber = 10;
-    public String someString = "Hello";
+    public int number = 10;
```

Comment:
Consider using a bulk rename feature in your IDE to rename variables consistently throughout the file. [medium]